src/mgr/PyModuleRegistry.h

```diff
@@ -151,7 +151,9 @@ class PyModuleRegistry
   void get_health_checks(health_check_map_t *checks);
 
   void get_progress_events(map<std::string,ProgressEvent> *events) {
-    return active_modules->get_progress_events(events);
+    if (active_modules) {
```

Comment:
Just to ensure, we don't want to `ceph_assert()` this, right?

Reply:
Yeah, the caller is send_report() and that happens when we're standby too.